### ISSUE TITLE
Possible to stream RTSP/RTMP?

### ISSUE THREAD

**Issue by ZebonianJustice** [rdp/screen-capture-recorder-to-video-windows-free#125]
Nice simple app, is there any posibillity to have RTSP or RTMP streaming added?
I have never used UDP stream and from what i understand i need to have a server to SEND my stream to? With RTSP i only need to have VLC playback my stream.

**Comment by rdp** [comment]
Yeah you should be able to just plug that into the stream url like rtmp://host:port etc...

**Comment by ZebonianJustice** [comment]
But in the description it says that i should put in a adress to some kind of server?
What adress should i put there if i would like to stream it?
Just my computers ip-adress? And then what port?

**Comment by rdp** [comment]
arbitrary port 127.0.0.1 host (then receive it with "whatever you want to
receive it with") FWIW good luck, it's not tested much! :)
